### ISSUE TITLE
Valdator 1.1.1 throws NullPointerException in Result.getReportDocument for malformed xml input

### ISSUE THREAD

**Issue by JannickWeisshaupt** [itplr-kosit/validator#40]
## Expected Behavior
The validator generates an appropriate report in the case of a malformed xml input file
## Actual Behavior
The respective instance of de.kosit.validationtool.api.Result returns null for result.getReport()
and throws a NullPointerException for result.getReportDocument().
## Examples
Any non xml document suffices as input.
Then
```
Result report = kositValidator.checkInput(input);
report.getReportDocument() 
```
throws NPE

**Comment by apenski** [comment]
fixed in branch [1.1.x](/itplr-kosit/validator/tree/branch-1.1.x). We will prepare a release package soon.
@JannickWeisshaupt thanks for reporting this issue. Feel free to test this fix

**Comment by apenski** [comment]
fixed
